### PR TITLE
notify-34 fix skipped tests

### PR DESCRIPTION
**Pull request by terrazoon**
https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=10024759

This PR reduces the number of skips from 120 to 34.  The remaining skips are more problematical and will take more time to resolve (inbound SMS, etc.)

### CODE REVIEW

**Comment by terrazoon** [comment]
https://github.com/orgs/GSA/projects/20/views/1?pane=issue&itemId=10024759

**Comment by terrazoon** [comment]
the x-fail feature is a new thing for me, but as I understand it it is meant to mark tests that are expected to fail -- due to a bug of some kind.  And that the long-term goal would be to have no x-fails.

However, at least in some cases here, people seem to be using as an alternative to 'pytest.raises' or similar, more common, syntax.  That just seems wrong to me, because then you have to manually evaluate every x-fail to see if it due to a bug or due to the developer just wanting to use an alternative syntax for a negative test.  

So I'm getting rid of x-fail if the test (correctly) fails and rewriting the test with more standard syntax.

File: tests/app/service_invite/test_service_invite_rest.py [hunk 1]
```diff
@@ -126,13 +124,12 @@ def test_create_invited_user_invalid_email(client, sample_service, mocker, fake_
...
 
 
-@pytest.mark.skip(reason="Needs updating for TTS: Failing for unknown reason")
 def test_get_all_invited_users_by_service(client, notify_db_session, sample_service):
     invites = []
     for i in range(0, 5):
         email = 'invited_user_{}@service.gov.uk'.format(i)
         invited_user = create_invited_user(sample_service, to_email_address=email)
-
+        print(f"INVITED USER = {invited_user.auth_type}")
```
Comment (stvnrlly) [inline tests/app/service_invite/test_service_invite_rest.py:132]: Can the print statements be removed?

**Review by stvnrlly — APPROVED** [review]

File: tests/app/service_invite/test_service_invite_rest.py (hunk 1 above)
Comment (terrazoon) [reply tests/app/service_invite/test_service_invite_rest.py]: will do

**Pull request merged**